demosplan/DemosPlanCoreBundle/Entity/User/Customer.php

```diff
@@ -186,18 +186,18 @@ class Customer extends CoreEntity implements UuidEntityInterface, CustomerInterf
      *
      * @var string
      *
-     * @ORM\Column(name="simple_language_overview_description", type="text", nullable=false, options={"default":""})
+     * @ORM\Column(name="simple_language_overview_description", type="text", nullable=false)
      */
     #[Assert\Length(max: 65536)]
     protected $overviewDescriptionInSimpleLanguage = '';
 
     public function __construct(/**
      * @ORM\Column(name="_c_name", type="string", length=50, nullable=false)
      */
-    private string $name, /**
+        private string $name, /**
      * @ORM\Column(name="_c_subdomain", type="string", length=50, nullable=false)
      */
-    private string $subdomain, string $mapAttribution = '')
+        private string $subdomain, string $mapAttribution = '')
```

Comment:
This indentation seems to be a mistake.

Reply:
I dont like what the style fixer does!
It wasn't me 4d62cca1cbf0553c05be7f209bee90ba2e429965